packages/menu/src/behaviors/MenuBehavior.js

```diff
@@ -108,11 +109,24 @@ export default class MenuBehavior extends Component {
   getOptionsInfo = () => this.state.optionInfo;
 
   setActiveOption = activeOption => {
-    if (this.props.onChange) {
-      this.props.onChange(activeOption);
+    const { onChange, unselect } = this.props;
+    const unselectPropValue = unselect === undefined ? true : unselect;
```

Comment:
I don't think you have to write it like this as `undefined` equaling `true` is a bit weird. I think you are trying to have it default to `undefined` and you can do that with `defaultProps`.